framework/console/controllers/MessageController.php

```diff
@@ -76,9 +76,12 @@ public function actionConfig($filePath)
      * @param string $configFile the path or alias of the configuration file.
      * You may use the "yii message/config" command to generate
      * this file and then customize it for your needs.
+     * @param string $languages comma-separated list of language's codes.
+     * This parameter override 'languages' array content
+     * from $configFile with specified language's codes.
      * @throws Exception on failure.
      */
-    public function actionExtract($configFile)
+    public function actionExtract($configFile, $languages = null)
```

Comment:
you can use an `array` typehint here to make the explode be done automatically by yii.
Here is an example of how this works: https://github.com/yiisoft/yii2-apidoc/blob/master/commands/ApiController.php#L42